settings.gradle

```diff
@@ -5,7 +5,8 @@ def modules = [
   'coral-spark',
   'coral-schema',
   'coral-spark-plan',
-  'coral-trino'
+  'coral-trino',
+  'shading'
```

Comment:
Shall we be more specific on what it shades, or are we foresee other shading components ?